packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -199,38 +614,27 @@ class DatePickerInput extends React.Component {
           name={name}
           value={dateValue}
         />
-        <Input
-          {...additionalInputProps}
-          className={inputClasses}
-          type="text"
-          name={'terra-date-'.concat(name)}
-          value={value}
-          onChange={this.handleOnChange}
-          placeholder={placeholder}
-          onFocus={onFocus}
-          onBlur={onBlur}
-          ariaLabel={value ? `${label}, ${getLocalizedDateForScreenReader(DateUtil.createSafeDate(dateValue), { intl: this.props.intl, locale: this.props.intl.locale })}` : label}
-        />
-        <Button
-          className={buttonClasses}
-          text={buttonText}
-          onClick={this.handleOnButtonClick}
-          onKeyDown={this.handleOnKeyDown}
-          icon={Icon}
-          isIconOnly
-          isCompact
-          isDisabled={additionalInputProps.disabled}
-          onBlur={onBlur}
-          onFocus={onButtonFocus}
-          refCallback={buttonRefCallback}
-        />
+        <VisuallyHiddenText text={value ? `${label}, ${getLocalizedDateForScreenReader(DateUtil.createSafeDate(dateValue), { intl, locale: intl.locale })}` : label} />
+        {dateInputFormat}
       </div>
-    );
-  }
-}
+      <Button
+        className={buttonClasses}
+        text={buttonText}
```

Comment:
```suggestion
        text={intl.formatMessage({ id: 'Terra.datePicker.openCalendar' })}
```

Reply:
This was a part of the previous implementation.
Changed here: https://github.com/cerner/terra-framework/pull/1208/commits/ba9d3c9b977ded21c1db0789cbe2b1e43dd3c7c6